<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiFileView</class>
 <widget class="QWidget" name="MultiFileView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="CompileWidget" name="compileWidget"/>
      <widget class="MultiFileHexView" name="tableView"/>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="FlameGraph" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>620</width>
         <height>68</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionCompile">
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CompileWidget</class>
   <extends>QWidget</extends>
   <header>compilewidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>MultiFileHexView</class>
   <extends>QTableView</extends>
   <header>multifilehexview.hpp</header>
  </customwidget>
  <customwidget>
   <class>FlameGraph</class>
   <extends>QWidget</extends>
   <header>flamegraph.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
